packages/roosterjs-editor-core/lib/editor/createEditorCore.ts

```diff
@@ -70,6 +86,8 @@ function calcDefaultFormat(node: Node, baseFormat: DefaultFormat): DefaultFormat
         bold: baseFormat.bold,
         italic: baseFormat.italic,
         underline: baseFormat.underline,
+        ogsb: baseFormat.ogsb,
+        ogsc: baseFormat.ogsc,

```

Comment:
ogsb/ogsc shouldn't be used in code -- it's extremely unfriendly to new readers.
Use descriptive names & we can uglify our bundles if we care about the bundle size impact